packages/presentation/lib/app/my_app.dart

```diff
@@ -46,14 +48,90 @@ class _MyAppState extends BlocScreenState<MyApp, AppBloc> {
           final appData = result.data?.data;
           if (appData is AppData) {
             return Scaffold(
-              body: Navigator(
-                onPopPage: (Route<dynamic> route, dynamic result) {
-                  bloc.handleRemoveRouteSettings(route.settings);
-                  return route.didPop(result);
-                },
-                pages: appData.pages.toList(),
+              body: Row(
+                children: [
+                  appData.hideNavBar ||
+                      !(Responsive.switchToSideBar(context))
+                      ? const SizedBox.shrink()
+                      : Container(
+                          padding: const EdgeInsets.only(bottom: Dimens.size14),
+                          decoration: const BoxDecoration(
+                            border: Border(
+                              top: BorderSide(
+                                width: Dimens.size05,
+                                color: PrimaryColors.primaryUnselected,
+                              ),
+                            ),
+                          ),
+                          child: NavigationRail(
+                            backgroundColor:
+                                PrimaryColors.primaryBackgroundColor,
+                            selectedIndex: bloc.selectedIndex,
+                            onDestinationSelected: (index) {
+                              bloc.onItemTapped(index);
+                            },
+                            labelType: NavigationRailLabelType.none,
+                            destinations: <NavigationRailDestination>[
+                              NavigationRailDestination(
+                                icon: SvgPicture.asset(
+                                  ImagesPath.movieIcon,
+                                  color: PrimaryColors.whiteWithOpacity80,
+                                ),
+                                selectedIcon: SvgPicture.asset(
+                                  ImagesPath.movieIcon,
+                                  color: PrimaryColors.primarySelected,
+                                ),
+                                label: const Text('Home'),
```

Comment:
Localize please.

Reply:
it is just stubs, they invisible for user

Reply:
labelType: NavigationRailLabelType.none is set, but labels still required, used for semantics

Reply:
Blind users can be affected

Reply:
fixed